app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt

```diff
@@ -162,6 +162,10 @@ class MagicFirebaseMessagingService : FirebaseMessagingService() {
                             cancelExistingNotificationWithId(applicationContext, signatureVerification!!.userEntity, notificationId)
                         } else if (deleteAll) {
                             cancelAllNotificationsForAccount(applicationContext, signatureVerification!!.userEntity)
+                        } else if (deleteMultiple) {
+                            notificationIds.forEach {
```

Comment:
I am not sure how json is handling this is if server response is empty/null.
Is notificationIds (long[]) then also null, then it will fail.
Or is deleteMultiple only true if there is really an array for ids?

Reply:
`deleteMultiple` is a boolean value within the json response, so I'd say "false if not present" so the loop won't be reached when the (also new) flag isn't there _and_ true.

Reply:
https://github.com/nextcloud/notifications/pull/939/files#diff-5f4fa2d752904832306c56f7832e2126a1e12db8dd1679abd71e526b77989821R544-R558

When it's empty it's delete-all, when its one entry its delete when it's multiple its delete-multiple

Reply:
Scenarios are
```
{"delete-all":true}
{"delete":true,"nid":1234}
{"delete-multiple":true,"nids":[1,2,3,4]}
```